sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkOptions.java

```diff
@@ -407,19 +407,19 @@
 
   void setQueryLanguage(String value);
 
-  @Description("Base name of Kafka source topic in streaming mode.")
+  @Description("Base name of Kafka events topic in streaming mode.")
   @Nullable
-  @Default.String("nexmark-source")
-  String getKafkaSourceTopic();
+  @Default.String("nexmark-events")
+  String getKafkaEventsTopic();
```

Comment:
Not sure if the new name sounds better. Normally just `kafkaTopic` is fine. Since there are are both sink and source topics, explicit source and sink sounds good. I would go with consistency between pubsub and Kafka topic option here.

Reply:
I see. Let's keep `kafkaTopic` and 'kafkaResultsTopic` then? Some how `kafkaEventsTopic` sounds a bit off :).